notebooks/book2/14/logreg_iris_bayes_2d_pymc.ipynb

```diff
@@ -0,0 +1,356 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        savefig("logreg_iris_bayes_2d", dpi=300)
</pre><p>avoid using <code>dpi</code>, maybe it can change figure dimensions.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>